bot.rb

```diff
@@ -14,12 +14,21 @@ class Bot < Summer::Connection
 
   BALL8_TIPS = YAML.load(File.read(File.join(File.dirname(__FILE__), 'config', '8ball.yml')))
   REASON_TIPS = YAML.load(File.read(File.join(File.dirname(__FILE__), 'config', 'reasons.yml')))
+  OPS = YAML.load(File.read(File.join(File.dirname(__FILE__), "config", "ops.yml")))
   CONFIG = YAML::load_file("config/summer.yml")
   def did_start_up
     ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'] || config['database'])
     privmsg("Bot started up at #{Time.now.strftime("%d %B %Y %H:%M")}", CONFIG['owner'])
     @pastebin_dumbass = {}
   end
+  
+  def ops_command(sender, channel, message, opts={})
+    if authorized?
+      direct_at(channel, "#{sender} has requested help from #{OPS[channel].join(", ")}.")
+    else
+      direct_at(channel, "I'm sorry #{sender}, but you need to be authenticated to use this service. Try authenticating with NickServ first.")
```

Comment:
If a firealarm doesn't do anything when you pull it, because you don't have a key, isn't that bad design?